sdk/python/examples/bayesianoptimization-katib-sdk.ipynb

```diff
@@ -0,0 +1,543 @@
+       "      'parameterType': 'int'}],\n",
+       "    'trialTemplate': {'goTemplate': {'rawTemplate': 'apiVersion: \"batch/v1\"\\nkind: Job\\nmetadata:\\n  name: {{.Trial}}\\n  namespace: {{.NameSpace}}\\nspec:\\n  template:\\n    spec:\\n      containers:\\n      - name: {{.Trial}}\\n        image: docker.io/kubeflowkatib/mxnet-mnist\\n        command:\\n        - \"python3\"\\n        - \"/opt/mxnet-mnist/mnist.py\"\\n        - \"--batch-size=64\"\\n        {{- with .HyperParameters}}\\n        {{- range .}}\\n        - \"{{.Name}}={{.Value}}\"\\n        {{- end}}\\n        {{- end}}\\n      restartPolicy: Never'}}},\n",
+       "   'status': {'completionTime': None,\n",
+       "    'conditions': [{'lastTransitionTime': '2020-03-04T10:46:49Z',\n",
+       "      'lastUpdateTime': '2020-03-04T10:46:49Z',\n",
+       "      'message': 'Experiment is created',\n",
+       "      'reason': 'ExperimentCreated',\n",
+       "      'status': 'True',\n",
+       "      'type': 'Created'}],\n",
+       "    'currentOptimalTrial': {'bestTrialName': '',\n",
+       "     'observation': {'metrics': None},\n",
+       "     'parameterAssignments': None},\n",
+       "    'startTime': '2020-03-04T10:46:49Z'}}],\n",
+       " 'kind': 'ExperimentList',\n",
+       " 'metadata': {'continue': '',\n",
+       "  'resourceVersion': '7035035',\n",
+       "  'selfLink': '/apis/kubeflow.org/v1alpha3/namespaces/anonymous/experiments'}}"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "kclient.get_experiment()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## get experiment status"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Running'"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "kclient.get_experiment_status(name=\"bayesianoptimization\", namespace=namespace)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Is experiment succeeded"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "False"
+      ]
+     },
+     "execution_count": 35,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "kclient.is_experiment_succeeded(name=\"bayesianoptimization\", namespace=namespace)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## delete experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " kclient.delete_experiment(name=\"bayesianoptimization\", namespace=namespace)"
```

Comment:
It is better to move `delete_experiment` to the latest command in example.
You are run few commands after deleting Experiment.

Reply:
done